src/Wallabag/CoreBundle/Resources/views/themes/material/Tag/tags.html.twig

```diff
@@ -13,7 +13,16 @@
         <ul class="card-tag-labels">
             {% for tag in tags %}
                 <li title="{{tag.label}} ({{ tag.nbEntries }})" id="tag-{{ tag.id }}">
-                    <a href="{{ path('tag_entries', {'slug': tag.slug}) }}" class="card-tag-link">{{tag.label}} ({{ tag.nbEntries }})</a>
+                    <a href="{{ path('tag_entries', {'slug': tag.slug}) }}" class="card-tag-link">
+                        {{tag.label}}&nbsp;({{ tag.nbEntries }})
```

Comment:
I know it was already there like this, but I think `{{ tag.label }}` (adding spaces around) might be better

Reply:
Ok, done 😄.